Data/Sys/GameSettings/GMSE01.ini

```diff
@@ -132,8 +95,86 @@ $D-Pad Left For Normal Mario
 4240E12C 4DE63F80
 4240E12C 4DE83F80
 00000000 40000000
-$Widescreen Heat Wave Fix [YoshiOG1]
-043AA050 3F2AAAAB
-043AA05C BF2AAAAB
-043AA060 3F2AAAAB
-043AA070 3FAAAAAB
+[Gecko]
+$Improved Jesus Mode DPad Up/Down = On/Off
```

Comment:
that was the original name in the INI and it wasnt my code either. I just converted it to a Gecko code so it would work on current dolphin builds